Components/Button/button.css

```diff
@@ -0,0 +1,91 @@
+.button-container {
+  background-color: var(--lightGray-color);
+  min-height: 20rem;
+  border: 1rem solid var(--lightGray-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  flex-wrap: wrap;
+}
+
+.btn {
+  border: none;
+  padding: 1rem;
+  text-align: center;
+  min-width: 10rem;
+  cursor: pointer;
+  font-weight: 600;
+  color: var(--white-color);
+  background-color: var(--primary-color);
+  border-radius: 0.3rem;
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-secondary {
+  background-color: var(--secondary-color);
+}
+
+.btn-error {
+  background-color: var(--danger-color);
+}
+
+.btn-success {
+  background-color: var(--success-color);
+}
+
+.outline {
+  background-color: var(--white-color);
+  color: var(--primary-color);
+  border: 1px solid var(--primary-color);
+}
+
+.outline:hover {
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.outline-secondary {
+  background-color: var(--white-color);
+  color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+}
+
+.outline-secondary:hover {
+  background-color: var(--secondary-color);
+  color: var(--white-color);
```

Comment:
The contrast between the background and the text is low, hindering the readability of secondary buttons. 

![image](https://user-images.githubusercontent.com/94738532/152998574-3d5c61a4-43d6-4e25-b275-1eebc5211032.png)

Reply:
As @SinchanaMS  said , you can either change the color of your secondary button or change the color of your background container. Rest is good:)